ledger-tool/src/main.rs

```diff
@@ -2244,6 +2254,15 @@ fn main() {
                     },
                 );
 
+                let snapshot_archive_format = arg_matches
```

Comment:
Btw, in case we have any metrics for snapshot, it might be a good idea to also include the format in the metrics (or even better, support group-by operation on compression).

Reply:
Yes. That will let us track and compare different compression format used in the cluster. I will add it to the report.